launcher/src/test/java/org/apache/spark/launcher/LauncherServerSuite.java

```diff
@@ -17,19 +17,42 @@
 
 package org.apache.spark.launcher;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
This might be your IDE, but please don't explode (or re-order) existing imports like this. It's unnecessary noise and actually against Spark's code conventions.